### ISSUE TITLE
Question about get_full_upload_url function

### ISSUE THREAD

**Issue by mittus** [bigin/ItemManager_2.0#14]
Hello!!

I was problem with image url's in file upload fields.

I need get absolute image urls without domain name. For that I added new input after Title input:
![screenshot_20](https://user-images.githubusercontent.com/711179/46460849-80e46c00-c7bd-11e8-9d17-2b78b6bc28f7.png)

But I seen bad urls: 
![screenshot_21](https://user-images.githubusercontent.com/711179/46461017-de78b880-c7bd-11e8-9c9f-9f00a8b4fef7.png)

Then I loock to the function in the UploadHandler.php and see that:
![screenshot_19](https://user-images.githubusercontent.com/711179/46461072-049e5880-c7be-11e8-9a1a-886a3f543df7.png)

When I use this short url, all functions on fileupload field work's correct:
![screenshot_22](https://user-images.githubusercontent.com/711179/46461145-45966d00-c7be-11e8-8cef-1ac74b3a1949.png)
Then I got truthful urls:
![screenshot_23](https://user-images.githubusercontent.com/711179/46461491-12a0a900-c7bf-11e8-811a-33cb7c68be8f.png)

I use short abs. urls anywhere, because I am working with git and has a problems with urls in local/production projects.
![screenshot_24](https://user-images.githubusercontent.com/711179/46461760-c99d2480-c7bf-11e8-9f61-14f0721a99cf.png)

Is it possible to take this into the plugin?

**Comment by bigin** [comment]
I can't remember why I did this. Did you tested it on install in a subdirectory, does it work?

**Comment by bigin** [comment]
If I remember correctly, the full URL is generated on runtime, is not static, and isn't stored anywhere. It's only used when displaying images via the file upload field, and does not depend on your environment.

**Comment by mittus** [comment]
> I can't remember why I did this. Did you tested it on install in a subdirectory, does it work?

Yes, with subdirectory my variant dosen't work. But I fix that with global variable `$SITEURL`, and now it work with all sites:
![screenshot_25](https://user-images.githubusercontent.com/711179/46481945-c4a69800-c7f4-11e8-95e2-0687f98bf32e.png)

> If I remember correctly, the full URL is generated on runtime, is not static, and isn't stored anywhere. It's only used when displaying images via the file upload field, and does not depend on your environment.

Yes, its true. But I will need get this url to the "input" on my project. I would copy this url, and pasting it, when it needed, into the wysiwyg redactor:
![screenshot_26](https://user-images.githubusercontent.com/711179/46482269-92496a80-c7f5-11e8-97ae-4be32d2798a4.png)
